doc/release-notes.md

```diff
@@ -73,6 +73,7 @@ RPC changes
 - The `fundrawtransaction` RPC will reject the previously deprecated `reserveChangeKey` option.
 - `sendmany` now shuffles outputs to improve privacy, so any previously expected behavior with regards to output ordering can no longer be relied upon.
 - The new RPC `testmempoolaccept` can be used to test acceptance of a transaction to the mempool without adding it.
+- The new RPC `clearmempool` can be used to force all mempool transactions to be removed.
```

Comment:
If it is only for testing, it shoulnd't be mentioned in the release notes?

Reply:
I don't know about the exact policy and included it here to have them "complete".  But of course I'm happy to remove the mention if it is common to not include changes that are not meant (mainly) for end users.

Reply:
Removed.